app/views/qsets/_question_page_header_large_screen.html.erb

```diff
@@ -9,9 +9,14 @@
     <%= render "all_mine_other_filter" %>
   </div>
 </div>
-<div class="row col-xs-12">
+<div class="row">
```

Comment:
Typically you'd also need a nested `<div class='col-...'>` inside the row. Otherwise it might behave erratically, or break if more things get stuffed into the row.